lib/modules/default.nix

```diff
@@ -4,5 +4,6 @@ rec {
   keyboard = import ./keyboard.nix { inherit lib; };
   neovim = import ./neovim.nix { inherit lib; };
   themes = import ./themes.nix { inherit lib neovim zsh; };
+  tmux = import ./tmux.nix { inherit lib; };
   zsh = import ./zsh.nix { inherit lib; };
 }
```

Comment:
are `tmuxModule`, `zshModule` and `neovimModule` used in so many different places that they needed extracting into `lib`?

Reply:
no just following the rest of the patterns.

Reply:
My understanding of this was that we would extract those only if they were to be used in several places, like themes which are used globaly and per-module. Extracting those makes it a bit harder to work on the module as the type definition is not directly in the same file.

Reply:
I agree with you but currently, most of these modules are scheduled for refactoring and I'm hoping we can remove them in favor of home-manager definition. I'm inclined to keep it as-is for now as the scope is already large enough. My next objective is Mac support, then re-do the template at which point I can figure out how to refactor these modules.